s3-import-commons/build.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
 
 test {
     environment "MAX_SLEEP_TIME", "1"
+    environment "IMPORT_EVENT_TYPE", "importEventTypeTestingValue"
```

Comment:
set env variable for testing. The env variable is read as a constant for simplicity, so we need to set it in the test environment.